test/relations.test.js

```diff
@@ -4987,7 +4987,7 @@ describe('relations', function() {
       Category.findById(category.id, function(err, cat) {
         if (err) return done(err);
         var link = cat.items.at(0);
-        link.updateAttributes({notes: 'Updated notes...'}, function(err, link) {
```

Comment:
> this solution implemented in this pr, which is to use updateById which is one of the methods of embedsMany to fix the situation

If I understand correctly, there are two apis:
- category.items.updateById
- item.updateAttributes

Probably they both can update the embedded instance, but this test case is supposed to test the second one, not "user has a way to update the embedded instance" so whether the first one works or not doesn't matter. 

Actually I am fine to skip this test case if it turns out the embedded instance update strategy has conflict with connector's update/updateAttributes function....